web/src/navbar_alerts.ts

```diff
@@ -283,12 +320,23 @@ export function initialize(): void {
         },
     );
 
-    $(".accept-bankruptcy").on("click", function (e) {
-        e.preventDefault();
-        const $process = $(this).closest("[data-process]");
-        show_step($process, 2);
-        setTimeout(unread_ops.mark_all_as_read, 1000);
-        $(window).trigger("resize");
+    $("#navbar_alerts_wrapper").on("click", ".accept_bankruptcy", function (this: HTMLElement) {
+        const $accept_button = $(this);
+        $accept_button.prop("disabled", true).css("pointer-events", "none");
+        const $banner = $(this).closest(".banner");
+        let count = 0;
+        const timer = setInterval(() => {
+            if (count > 100) {
+                clearInterval(timer);
+                return;
+            }
+            banners.set_banner_progress($banner, count, 100);
+            count += 25;
+        }, 300);
+        unread_ops.mark_all_as_read();
+        setTimeout(() => {
+            banners.close($banner);
+        }, 2000);
```

Comment:
So this looks like a basically fraudulent progress bar -- it just advances 25% every 300ms, while doing the actual operation in parallel.

I don't think that's a good idea. I'll see if you fix it later in the PR, but in general this sort of thing must be called out in the commit message.  I don't really see why this wasn't its own commit, actually.

Reply:
Looks like this remains. I think probably we should just remove the progress bar from this PR, and plan to think about that as a follow-up? It seems not essential to the goals of this PR, and it might not be trivial to hook it up to the right data, so it should get its own development and review space aka PR.

Reply:
Yep, the progress bar wasn't really an accurate depiction of the actual operation, but instead something to show for the 2000ms timeout before the operation is actually performed.

Reply:
I have removed the progress bar and its application from the PR. Makes sense to implement it as a follow-up PR.